examples/siro_sandbox/sandbox_app.py

```diff
@@ -163,6 +172,13 @@ def on_environment_reset(self):
         ]
         self._goal_positions = [mn.Vector3(pos) for pos in goal_positions_np]
 
+    @property
+    def task_complete(self):
+        config = self.env._config
```

Comment:
can you cache `success_measure_name` on startup? Accessing `config` is apparently super-slow and should generally be avoided per step.